src/golang/cmd/migrator/versions/000011_exec_state_column_backfill/queries.go

```diff
@@ -0,0 +1,68 @@
+package _000011_exec_state_column_backfill
+
+import (
+	"context"
+
+	"github.com/aqueducthq/aqueduct/lib/collections/shared"
+	"github.com/aqueducthq/aqueduct/lib/collections/utils"
+	"github.com/aqueducthq/aqueduct/lib/database"
+	"github.com/google/uuid"
+)
+
+type opResultWithMetadata struct {
+	Id       uuid.UUID              `db:"id" json:"id"`
+	Status   shared.ExecutionStatus `db:"status" json:"status"`
+	Metadata NullMetadata           `db:"metadata" json:"metadata"`
+}
+
+type opResultWithExecState struct {
+	Id        uuid.UUID                 `db:"id" json:"id"`
+	Status    shared.ExecutionStatus    `db:"status" json:"status"`
+	ExecState shared.NullExecutionState `db:"exec_state" json:"exec_state"`
```

Comment:
Won't this no longer work if we modify `shared.NullExecutionState` in the future? This is why we have to redefine the structs to the state they were in at the time of this migration.

Reply:
Good point, fixed